src/main/update_installer.js

```diff
@@ -0,0 +1,40 @@
+import {dialog, shell} from "electron";
+import phin from "phin";
+const semverGreaterThan = require("semver/functions/gt");
+const {version} = require("../../package.json");
+
+const getJSON = phin.defaults({method: "GET", parse: "json", headers: {"User-Agent": "BetterDiscord Installer"}});
```

Comment:
Can you add the `followRedirects: true` option for future-proofing with GitHub's API fun?